src/contrib-doc.md

```diff
@@ -44,18 +44,51 @@ cargo doc --all --no-deps
 cargo deadlinks --dir target/doc
 ```
 
-TODO: document how to make cross-links
-
 Rand API docs are automatically built and hosted at
-[rust-random.github.io/rand/](https://rust-random.github.io/rand/) for the
-latest code in master.
+[rust-random.github.io/rand] for the latest code in master.
 
 ### Getrandom crate
 
 The [rust-random/getrandom](https://github.com/rust-random/getrandom)
 repository contains only a single crate, hence a simple `cargo doc` will
 suffice.
 
+### Cross-crate links
+
+When referring to another crate, we prefer linking to the crate page on
+crates.io since (a) this includes the README documenting the purpose of the
+crate and (b) this links directly to both the repository and the API
+documentation. Example:
+
+```rust
+// Link to the crate page:
+//! [`rand_chacha`]: https://crates.io/crates/rand_chacha
+```
+
+When referring to an item from within another crate,
+
+1.  if that item is accessible via a crate dependency (even if not via the
+    public API), use the Rust item path
+2.  when linking to another crate within the `rust-random/rand` repository,
+    relative paths within the generated documentation files (under `target/doc`)
+    can be used; these work on [rust-random.github.io/rand] but not
+    currently on `docs.rs` (see [docs#204])
+3.  if neither of the above are applicable, use an absolute link
```

Comment:
Do you mean an absolute link to docs.rs, like in the example? Maybe mention that here, including that the patch version should be omitted.

Reply:
I just ended up using absolute links for #10; this seemed easiest.

Unfortunately any links to `docs.rs/rand/*/...` are currently broken since the latest build failed. I'm not so sure we should rely on it? ([Reported here](https://github.com/rust-lang/docs.rs/issues/369).)

Reply:
The absolute links are a bit problematic, because they link to master, and not to the correct release.

Reply:
The contents of the book aren't versioned either. (For the most part they still apply to old versions, but not entirely.)